<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>375</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icons/icons/WiNGSPAN Alliance Logo.ico</normaloff>:/icons/icons/WiNGSPAN Alliance Logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="kmGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>581</width>
      <height>71</height>
     </size>
    </property>
    <property name="title">
     <string>Enter External Killmail URL</string>
    </property>
    <widget class="QLineEdit" name="kmLineEdit">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>25</y>
       <width>461</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string extracomment="Test"/>
     </property>
    </widget>
    <widget class="QLabel" name="kmStatusLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>KM STATUS TEXT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getKMInfoButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>25</y>
       <width>106</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Fetch killmail info</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="locationGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>581</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Location will be auto-filled into the receipt when &amp;quot;Generate Delivery Receipt&amp;quot; is pressed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="title">
     <string>Enter Location</string>
    </property>
    <widget class="QLabel" name="locationLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>29</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>We met at </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="locationLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>506</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <zorder>locationLineEdit</zorder>
    <zorder>locationLabel</zorder>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>10</y>
      <width>344</width>
      <height>115</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/icons/icons/WiNGSPAN Title.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="generateReceiptButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>300</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Receipt</string>
    </property>
   </widget>
   <widget class="QLabel" name="receiptStatusLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>300</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RECEIPT STATUS TEXT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>logoLabel</zorder>
   <zorder>locationGroupBox</zorder>
   <zorder>kmGroupBox</zorder>
   <zorder>generateReceiptButton</zorder>
   <zorder>receiptStatusLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
